packages/ui/src/stories/foundation/typography.stories.tsx

```diff
@@ -91,3 +92,41 @@ export const BodyFont = () => {
     </Table>
   );
 };
+
+/**
+ * Fonts weight
+ */
+export const FontWeight = () => {
+  const fontWeight = [
+    { className: 'typography-brand-7 font-light', weight: '300' },
+    { className: 'typography-brand-7 font-normal', weight: '400' },
+    { className: 'typography-brand-7 font-medium', weight: '500' },
+    { className: 'typography-brand-7 font-semibold', weight: '600' },
+    { className: 'typography-brand-7 font-bold', weight: '700' },
+  ];
+  return (
+    <Table>
+      <Table.Caption>Font Weight</Table.Caption>
+      <Table.Header>
+        <Table.HeaderRow>
+          <Table.HeaderCell>Token</Table.HeaderCell>
+          <Table.HeaderCell>Value</Table.HeaderCell>
+          <Table.HeaderCell>Tailwind class</Table.HeaderCell>
+        </Table.HeaderRow>
+      </Table.Header>
+      <Table.Body>
+        {fontWeight.map(i => (
+          <Table.Row key={i.className}>
+            <Table.Cell>
+              <span className={i.className}>
+                {capitalizeFirstLetter(i.className.replace('typography-brand-7 font-', ''))}
```

Comment:
You could simply use CSS instead of JS, https://tailwindcss.com/docs/text-transform

Reply:
I see. That is a better approach I will role back the changes. Thanks Samitha